src/ReadableCollection.php

```diff
@@ -229,11 +229,11 @@ public function findFirst(Closure $p);
      * Applies iteratively the given function to each element in the collection,
      * so as to reduce the collection to a single value.
      *
-     * @psalm-param Closure(TReturn|TInitial|null, T):(TInitial|TReturn) $func
-     * @psalm-param TInitial|null $initial
+     * @psalm-param Closure(TReturn|TInitial, T):TReturn $func
+     * @psalm-param TInitial $initial
```

Comment:
If initial is not null, the closure does not need to accept `null` value for first param.